README.md

```diff
@@ -1,65 +1,43 @@
 # Introduction to deep learning
+This lesson gives an introduction to deep learning.
 
-[![Create a Slack Account with us](https://img.shields.io/badge/Create_Slack_Account-The_Carpentries-071159.svg)](https://swc-slack-invite.herokuapp.com/)
+## Lesson Design
+The design of this less can be found in the [lesson design](_extras/design.md)
```

Comment:
There is a typo:

The design of this lesson can be found

(and not less)

Reply:
fixed, thank you.